app/bundles/LeadBundle/Translations/en_US/messages.ini

```diff
@@ -416,5 +417,40 @@ mautic.lead.campaign.event.field="Contact Field"
 mautic.lead.campaign.event.field_descr="Select the contact field which holds the value you want to compare."
 mautic.maintenance.visitors="Visitors"
 mautic.report.group.contacts="Contacts"
-mautic.lead.import.label.stage="Stage"
 mautic.lead.exists.by.field="A contact with this value already exists"
+mautic.lead.field.object="Object"
+mautic.companies.menu.index="Companies"
+mautic.companies.menu.root="Companies"
+mautic.company.action.noresults.tip="Create a company to group contacts that belong to a same company. It will also help you keep important information about different companies you target."
+mautic.company.menu.new="New Company"
+mautic.company.menu.edit="Edit Company"
+mautic.company.name="Company name"
+mautic.company.company.number="Company number"
+mautic.company.company.source="Company source"
+mautic.company.annual.revenue="Annual revenue"
+mautic.company.email="Company email"
+mautic.company.website="Company website"
+mautic.company.phone="Company phone"
+mautic.company.fax="Company fax"
+mautic.company.numberOfEmployees="Number of employees"
+mautic.company.score="Company score"
+mautic.company.address1="Address line 1"
+mautic.company.address2="Address line 2"
+mautic.company.city="City"
```

Comment:
Consider using `transConditional()` to prevent from having to duplicate all these same translation strings. 

```$translator->transConditional("mautic.lead.field.{$field}", "mautic.company.{$field}")```

That will use mautic.lead.field.city if it exists otherwise use mautic.company.city. Goal is to reduce all these duplicate translation strings.